### ISSUE TITLE
How to use pre-trained weights ?

### ISSUE THREAD

**Issue by vlarmet** [Orion-AI-Lab/EfficientBigEarthNet#1]
Hi,

First of all, thank you for this repo!
I would like to use pre-trained weights for ResNet50 architecture in tensorflow. I have downloaded corresponding pre-trained weights in dropbox.

How should I do this ?
Build a ResNet50 and assign weights ?  

```
wts = "./checkpoint_ResNet50/checkpoints-3"
model = ResNet50(
            include_top=True,
            weights=None,
            input_shape=(120, 120, 12),
            pooling="avg",
            classes=43
)
model.load_weights(wts)

```  

I got about 3000 lines of warnings like these : 

```
WARNING:tensorflow:Unresolved object in checkpoint: (root).model
WARNING:tensorflow:Unresolved object in checkpoint: (root).optimizer
WARNING:tensorflow:Unresolved object in checkpoint: (root).save_counter
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer-0
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer-1
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer-2
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer-3
```
and finally an error : `AssertionError: Nothing except the root object matched a checkpointed value. Typically this means that the checkpoint does not match the Python program. The following objects have no matching checkpointed value`

**Comment by azavras** [comment]
Hi @vlarmet,
It seems that you are trying to load our custom ResNet50 pre-trained weights to a ResNet50 tensorflow.keras model. You can find the correct weights for your use case [here](https://www.dropbox.com/s/trsojsbjvlpt9jz/checkpoint_resnet50.zip?dl=0). 

Please keep in mind that during pre-training we use the 19-class nomenclature and that in addition to the 10th spectral band of Sentinel-2, which was also excluded in the original BigEarthNet implementation, we also excluded the 1st and 9th spectral bands as they do not contain information regarding the Earth’s surface.

Summing up, you should download the correct checkpoint files using the link mentioned above, set the input shape to (120, 120, 10) and set the classes argument equals to 19.

**Comment by vlarmet** [comment]
Thank you, I managed to do what I want !
